packages/core/crates/core-types/src/channels.rs

```diff
@@ -138,17 +146,30 @@ impl ChannelEntry {
     }
 }
 
-impl std::fmt::Display for ChannelEntry {
-    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        f.debug_struct("ChannelEntry")
-            .field("source", &self.source.to_string())
-            .field("destination", &self.destination.to_string())
-            .field("balance", &format!("{}", self.balance))
-            .field("ticket_index", &self.ticket_index.to_string())
-            .field("status", &self.status.to_string())
-            .field("channel_epoch", &self.channel_epoch.to_string())
-            .field("closure_time", &self.closure_time.to_string())
-            .finish()
+impl ChannelEntry {
```

Comment:
why don't merge this with previous `impl ChannelEntry`?

Reply:
The previous one has `#[cfg_attr(feature = "wasm", wasm_bindgen::prelude::wasm_bindgen)]` on it, so this one is wasm-incompatible, whereas the previous one is.